docs/setting-up-and-running.asciidoc

```diff
@@ -24,6 +25,21 @@ You can change the defaults by supplying a different address on the command line
 ./apm-server -e -E output.elasticsearch.hosts=ElasticsearchAddress:9200 -E apm-server.host=localhost:8200
 ----------------------------------
 
+[float]
+[[running-deb-rpm]]
+=== Debian Package / RPM
+
+The Debian package and RPM installations of APM Server create an `apm-server` user.
+To start the APM Server in this case, run:
+
+[source,bash]
+----------------------------------
+sudo -u apm-server apm-server -e [FLAGS]
```

Comment:
I got the example in the issue wrong, the right notation would be 
`sudo -u apm-server apm-server [<argument...>]`
you might want to scan trough docs and make sure we use the notation correctly everywhere, and consistently.

ref: http://docopt.org/

Reply:
Fixed. I might ping you tomorrow to have a quick chat about this. Are you saying `[<argument...>]` should replace `[FLAGS]` everywhere in the docs?